doc/content/getting-started/aws/ecs/prerequisites/_index.md

```diff
@@ -14,7 +14,7 @@ Before you can start the deployment, you will need to make sure that you have:
 - An SSH keypair registered in the AWS region of choice.
 - A domain name with a hosted zone in AWS Route 53. For multi-tenant deployments, you need to be able to add a wildcard subdomain under this domain.
 - A username and password for an SMTP server for sending emails, or a Sendgrid API key with "Mail Send" rights.
-- A LoRa Alliance NetID and DevAddr block. For development and testing it is possible to use NetID `000000` and DevAddr block `00000000/6`.
+- A LoRa Alliance NetID and DevAddr block. For development and testing it is possible to use NetID `000000` and DevAddr block `00000000/7`.
```

Comment:
To be completely correct, it would be possible to use NetID `000000` with DevAddr block `00000000/7` or NetID `000001` with DevAddr block `02000000/7`